### ISSUE TITLE
Investigate VoiceOver behavior for FAR search parameter summary

### ISSUE THREAD

**Issue by oddball-lindsay** [department-of-veterans-affairs/va.gov-team#71270]
From Jamie:
> when using VoiceOver (macOS screen reader), when I go line by line on the page after loading up search results, the “1 - 10 results for X” line reads very strangely, and I have no idea why.
> SK, another CAIA a11y, can’t duplicate this. hence it’s probably a fluke! but for all i know, governance will have a similar issue, so i wanted to flag it.

Watching the video, it seems to say "level one" between words

[Private Zenhub Video](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNzAwQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--eeeb78efdaad5d9611e78633348ddd9cef4c77f7/video1237180099.mp4)

Original message in Slack: https://dsva.slack.com/archives/C05L6HSJLHM/p1701786594842099?thread_ts=1701271694.454669&cid=C05L6HSJLHM

**Comment by oddball-lindsay** [comment]
Unable to reproduce, suspecting this is related to VoiceOver settings, perhaps Verbosity settings
